src/modules/comment_api.js

```diff
@@ -0,0 +1,34 @@
+const commentApi = (e) => {
+  const involmentAppID = 'hrqIdiXTh94rmLQMrXcG';
+  const index = Number(e.target.dataset.id);
+  const date = new Date();
+  const month = ['01', '02', '03', '04', '05', '06', '07', '08', '09', '10', '11', '12'];
+  const num = month[date.getMonth()];
+  const actualDate = `${date.getFullYear()}-${num}-${date.getDate()}`;
+  const name = e.target.parentElement.children[0].children[1].value;
+  const comment = e.target.parentElement.children[1].children[0].value;
```

Comment:
I recomend to make your functions more re-usable.
so that when you need to implement your function in another part of the project or maybe in another project, you can easily use the same function.